articles/multifactor-authentication/yubikey.md

```diff
@@ -194,47 +200,41 @@ return function (context, req, res) {
 }
 ```
 
-> Full source code is available [here](https://github.com/auth0/rules/blob/master/redirect-rules/yubico-mfa.md).
-
-This sample, uses a single Webtask to handle 3 states:
-
-* __Rendering the UI__ for the first time (implemented in the `otpForm` function).
-* __Capturing the YubiKey-NEO__ code and validating it with the Yubico API.
-* __Returning the result to Auth0__. If validation succeeds, the result is returned to Auth0 to continue with the login transaction.
+**NOTE:** The redirect to Auth0 contains two querystring parameters: `id_token` and `state`. `id_token` is a convenient and secure way of transferring information back to Auth0. `state` is mandatory to protect against CSRF attacks.
 
-> Notice the redirect back to Auth0 carries two query string values: an `id_token` and a `state` parameters. `id_token` is a convenient, and secure way of transferring information back to Auth0. `state` is mandatory to play back to protect against CSRF attacks.
-
-Notice __no secrets are hardcoded__ in the Webtask code. They are referred to by the variables `context.data.yubico_clientid` and `context.data.yubico_secret`. When the Webtask is created, these sensitive parameters are embedded in the Webtask token securely.
-
-Save the code above in a file somewhere in your file system. This tutorial assumes `yubico-mfa-wt.js` for the file.
+**NOTE:** No keys are hard-coded into the Webtask code. They are referred to by the variables `context.data.yubico_clientid` and `context.data.yubico_secret`. These parameters are securely embedded in the Webtask token when the Webtask is created.
 
 ### 1. Initialize Webtask CLI
 
-Code in __rules__ is automatically packaged as Webtasks by Auth0. Because this is a custom Webtask, it needs to be created with the Webtask CLI.
+*Rules* code is automatically packaged as Webtasks by Auth0. Since this is a custom Webtask, it must be created with the Webtask CLI.
 
-All instructions are available under __Account Settings__ on the [dashboard](${uiURL}/#/account/webtasks).
+Follow the instructions for installing Webtask CLI under [Account Settings > Webtasks](${uiURL}/#/account/webtasks) on the Auth0 dashboard.
 
 Once the Webtask CLI is installed, run:
 
-```
+```txt
 wt create --name yubikey-mfa --secret yubikey_secret={YOUR YUBIKEY SECRET} --secret yubikey_clientid={YOUR YUBIKEY CLIENT ID} --secret returnUrl=https://${account.namespace}/continue --output url --profile {WEBTASK PROFILE} yubico-mfa-wt.js
 ```
 
-The `WEBTASK PROFILE` is the value obtained from the __Account Settings__ above.
+The `WEBTASK PROFILE` is the value obtained from the Account Settings page.
```

Comment:
I dont see where the WEBTASK PROFILE value is indicated in the Account Settings page.